pyproject.toml

```diff
@@ -17,7 +17,7 @@ coverage_comment = 'coverage_comment.main:main'
 
 [tool.poetry.dependencies]
 python = "^3.10"
-coverage = "*"
+coverage[toml] = "*"
```

Comment:
See two lines below for the correct syntax :)

Reply:
Thanks. I've primarily used `setup.cfg` and hence not familiar entirely with pyproject.toml syntax. I'm still not sure if this will fix the problem (my CI still pulls your `latest` version instead of my branch)